nwb_conversion_tools/datainterfaces/blackrockdatainterface.py

```diff
@@ -94,17 +105,23 @@ def run_conversion(self, nwbfile: NWBFile, metadata: dict = None, use_times: boo
         stub_test: bool, optional (default False)
             If True, will truncate the data to run the conversion faster and take up less memory.
         """
-        if self.source_data['nsx_to_load'] != 6:
-            write_as_lfp = True
+        if self.source_data['filename'].split('.')[-1][-1] == '6':
+            write_as = 'raw'
+        elif write_as not in ['processed', 'lfp']:
+            write_as = 'processed'
+
+        print(f'Converting Blackrock {write_as} traces...')
```

Comment:
Are you 100% sure that this is a general feature of the Blackrock format and not just something specific to the Movshon labs way of formatting filenames? In general, I don't see a problem having `write_as` specified in the `conversion_options` rather than a lower level auto-configuration, but not a fan of overriding `run_converion` if it can ever be avoided..

Reply:
https://github.com/NeuralEnsemble/python-neo/blob/072adfdaeffcae3b4f2bc4e8eb0be971a38874c5/neo/rawio/blackrockrawio.py#L24-L42